src/OpenTelemetry.Instrumentation.EventCounters/EventCountersMetrics.cs

```diff
@@ -46,6 +46,7 @@ internal sealed class EventCountersMetrics : EventListener
     /// <param name="options">The options to define the metrics.</param>
     public EventCountersMetrics(EventCountersInstrumentationOptions options)
     {
+        _ = EventCountersInstrumentationEventSource.Log;
```

Comment:
Need to add comments describing why this is required.

Reply:
+1, also consider making it static (e.g. in cctor).